tests/config/unit.tests.yaml

```diff
@@ -162,6 +162,11 @@ sub:
   values:
   - 6.2.3.4.
   - 7.2.3.4.
```

Comment:
We may want to add a `NS` for `subzone.unit.tests.` here to make sure that a child works in addition to grandchild. Probably should have been here in the first place, but since we're in the neighborhood it's a good time to add it.